src/views/wiki/edition/OutingEditionView.vue

```diff
@@ -396,7 +400,9 @@ export default {
         this.document.date_end = this.document.date_start;
       }
     },
-
+    setCurrentDate() {
+      this.currentDate = new Date().toISOString().substring(0, 10)
```

Comment:
This conversion to stick to a date (which you only copied) is a bit ugly IMHO.
We use `date-fns` on c2c_ui, so using `format(new Date(), 'yyyy-MM-dd')` would seem more appropriate IMHO.

(applies also to line 246)

Reply:
Oh I did not know you were using date-fns. Way better indeed :)